docs/usage.md

```diff
@@ -0,0 +1,354 @@
+# Dusty Cli Usage
+
+## Options
+
+#### -d
+```
+Listen for user commands to Dusty
+
+Usage:
+  dusty -d [--suppress-warnings] [--preflight-only]
+
+Options:
+  --suppress-warnings  Do not display run time warnings to the client
+  --preflight-only  Only run the preflight_check, then exit
+```
+
+Runs the Dusty daemon.  This action must be performed as root.
+If you use Dusty's install script, a plist will be setup to run
+this daemon automatically for you.
+
+#### -v
+
+Prints the version of both the Dusty client and daemon
```

Comment:
This isn't accurate, just kill this and Zach can add a proper docstring when his thing works